pkg/cloudprovider/providers/azure/azure_backoff.go

```diff
@@ -396,8 +396,8 @@ func processRetryResponse(resp autorest.Response, err error) (bool, error) {
 		// suppress the error object so that backoff process continues
 		return false, nil
 	}
-	// Fall-through: stop periodic backoff, return error object from most recent request
-	return true, err
+	// Fall-through: stop periodic backoff
+	return true, nil
```

Comment:
@brendandburns is there any explaination for this? bug fix?

Reply:
@andyzhangx Yep a bug fix. Retry cases has been processed by `shouldRetryAPIRequest`, here should just stop backoff.